set.mm

```diff
@@ -46248,7 +46353,7 @@ conditioning it (with ` x e. z ` ) so that it asserts the existence of a
     $( Alternate proof of ~ axnul , proved from propositional calculus,
        ~ ax-gen , ~ ax-4 , ~ sp , and ~ ax-rep .  To check this, replace ~ sp
        with the obsolete axiom ~ ax-c5 in the proof of ~ axnulALT and type the
-       Metamath program "MM> SHOW TRACE_BACK axnulALT / AXIOMS" command.
+       Metamath program "MM> SHOW TRACE axnulALT / AXIOMS" command.
```

Comment:
The command is "show trace_back", but "show trace" works as abbreviation.

Reply:
Indeed. I would prefer to keep the full command, but on the webpage it appears as in index like TRACE<sub>BACK </sub> which is why I made that modification.

Reply:
You are right, and there are several MINIMIZE_WITH in set.mm which are displayed in the same ugly way. In 2019, I proposed that two consecutive underscores `__` should always be treated as literal `_`   (see google group https://groups.google.com/g/metamath/c/r3pwUvhI3NQ/m/fcEmU5OkAwAJ ). I do not know, however, if this was ever implemented. Currently, there aren't any two consecutive underscores `__`  in comments of set.mm.